cypress/integration/view-and-manage-evidence.spec.ts

```diff
@@ -42,8 +42,29 @@ describe('Can view and manage evidence', () => {
     cy.get('button').should('contain', 'Request new file');
 
     cy.get('h2').should('contain', 'Preview');
+    cy.get('img').should('be.visible');
+
     cy.get('figure').should('contain', 'PNG');
     cy.get('figure').should('contain', '24.7 KB');
+    cy.get('a').should('contain', 'Download').and('have.attr', 'href');
+
+    // TODO: uncomment when History is implemented
+    // cy.get('h2').should('contain', 'History');
+  });
+
+  it('lets you see an PDF document detail page with actions and information', () => {
+    cy.get(':nth-child(2) > :nth-child(2) > .lbh-heading-h3 > .lbh-link')
```

Comment:
Can we use something like an id (or other identifier, unrelated to the hierarchy) here instead of the path to the element? So in case we add other elements on the page (that change the hierarchy of the elements) we don't break the test.

Reply:
Good spot - I introduced `toReview` and `reviewed` div tags so we can be more specific when we try and find the links